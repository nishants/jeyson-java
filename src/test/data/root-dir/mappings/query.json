{
  "request": {
    "method": "GET",
    "urlPattern": "/query?.*"
  },
  "response": {
    "status": 200,
    "bodyFileName": "query.json"
  }
}