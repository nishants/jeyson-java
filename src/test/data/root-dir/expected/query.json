{"data":{"queries":{"time":"now","message":"hi"}}}